docs/intro/overrides.rst

```diff
@@ -100,42 +129,156 @@ The code above declares that:
 
 .. tip::
 
-    The URL patterns declared in the ``@handle_urls`` annotation can still be
+    The URL patterns declared in the ``@handle_urls`` decorator can still be
     further customized. You can read some of the specific parameters in the
     :ref:`API section <api-overrides>` of :func:`web_poet.handle_urls`.
 
+Item Class
+~~~~~~~~~~
+
+An alternative approach for the Page Object Overrides example above is to specify
+the returned Item Class. For example, we could change the previous example into
+the following:
+
+
+.. code-block:: python
+
+    from web_poet import handle_urls, WebPage, Returns
+
+
+    class GenericProductPage(WebPage, Returns[Product]):
+        def to_item(self) -> Product:
+            return Product(product_title=self.css("title::text").get())
+
+
+    @handle_urls("example.com", to_return=Product)
+    class ExampleProductPage(WebPage):
+        def to_item(self) -> Product:
+            ...  # more specific parsing
+
+
+    @handle_urls("anotherexample.com", to_return=Product, exclude="/digital-goods/")
+    class AnotherExampleProductPage(WebPage):
+        def to_item(self) -> Product:
+            ...  # more specific parsing
+
+
+    @handle_urls(["dualexample.com/shop/?product=*", "dualexample.net/store/?pid=*"], to_return=Product)
+    class DualExampleProductPage(WebPage):
+        def to_item(self) -> SimilarProduct:
+            ...  # more specific parsing
```

Comment:
I think the main and promoted way of using this feature should be this:

```py
    @handle_urls("example.com")
    class ExampleProductPage(WebPage[Product]):
        def to_item(self) -> Product:
            ...  # more specific parsing
```

Using to_return argument directly is only for advanced use cases; it should be discouraged.

Reply:
Updated in https://github.com/scrapinghub/web-poet/pull/84/commits/52a6f474bd97d1626eaf153241496a7a78c40df6 where it de-emphasizes this approach